electron/main.js

```diff
@@ -311,7 +311,8 @@ const createMainWindow = async () => {
 
     try {
       await scraper.login(username, password, email);
-      return { success: true };
+      const cookies = await scraper.getCookies();
+      return { success: true, cookies };
```

Comment:
Does the call to scraper.login always use the password? I think we first should check whether we already have the cookies.